### ISSUE TITLE
Yandex.Bar addon doesn't work

### ISSUE THREAD

**Issue by Kyogre** [BrowserWorks/waterfox#176]
Yandex.Bar extension (download here https://addons.mozilla.org/ru/firefox/addon/yandexbar/ ) doesn't work at all on Waterfox, but works fine on official Firefox 55.0.3 (tested on current and new clean profiles). In it's settings (opened through addons list) it displays this error:

XML Parsing Error: undefined entity
Location: chrome://yasearch/content/preferences/preferences.xul
Line Number 12, Column 1:<prefwindow buttonlabelextra2="&prefpane.widgets.extra2.button;" buttons="accept,cancel,extra2" height="620" id="&app.name;-prefwindow" minheight="605" minwidth="670" ondialogextra2="Preferences.doCommand('cmd_restoreDefault');" onunload="Preferences.onUnload();" persist="width height screenX screenY" title="&pref.window.title;" type="prefwindow" width="740" windowtype="&app.name;:Preferences" xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul">
^

**Comment by grahamperrin** [comment]
Is the issue reproducible with more recent version 8.23.2 with Waterfox 56.0.4.1?

**Comment by Kyogre** [comment]
1. I forgot to mention that this error appears only in addon version 8.21.2 (and older). If you compare version 8.21.2 and latest 8.23.2 version you will figure out that they are completely different addons in terms of functionality (8.21.2 is a featurefull toolbar, 8.23.2 is just an email addon), so I am talking exactly about good old version 8.21.2 in this bugreport.
2. Error don't appear in official Firefox 55.0.3 (or older), but appears in all Waterfox versions (tested now on 56.0.4.1, and some time ago I also tested on Waterfox versions 55.x)
3. Full error message and some more details I wrote in thread https://www.reddit.com/r/waterfox/comments/7dc4u5/yandexbar_addon_doesnt_work_in_waterfox_at_all/

**Comment by grahamperrin** [comment]
OK, https://addons.mozilla.org/addon/yandexbar/versions/?page=1#version-8.21.2

Compatibility reports for Firefox 56.0: 

- https://addons.mozilla.org/compatibility/reporter/yasearch%40yandex.ru?appver=56.0

– three of four users of 8.21.2 reported problems with 56.x so maybe it's a problem with the add-in, not with Waterfox. Have you sought advice from the developer of the extension?

**Comment by Kyogre** [comment]
I opened this bugreport when Waterfox 55.x was the latest one, so I wished that waterfox's developers will fix this error for Waterfox 55, not 56. I already know that yandexbar 8.21.2 doesn't work even on official Firefox since version 56.x, so I don't know what Waterfox devs can do in current situation.
In other words: error is relevant, but fix for Waterfox 56 is irrelevant, because impossible.

**Comment by Kyogre** [comment]
I opened this bugreport when Waterfox 55.x was the latest one, so I wished that waterfox's developers will fix this error for Waterfox 55, not 56 (because, as I said: in that time yandexbar addon 8.21.2 worked successfully in Firefox 55, but didn't work in Waterfox 55, so the problem was not in addon, but in Waterfox). I already know that yandexbar 8.21.2 doesn't work even on official Firefox since version 56.x, so I don't know what Waterfox devs can do in current situation.
In other words: the error is relevant, but fix for Waterfox 56 is irrelevant, because nearly impossible, I think.
Maybe you can inspect why 8.21.2 works in Firefox 55, but don't work in Waterfox 55 and then research a fix for Waterfox 56.

**Comment by grahamperrin** [comment]
> … when Waterfox 55.x was the latest …

Without looking at code, 

> Waterfox 55.2.0 … Patched security issues from …56 Release. …

– from [Waterfox 55 Release (Windows, Mac, Linux and Android)](https://www.waterfoxproject.org/blog/waterfox-55.0-release-download).

I see that version 8.21.2 of the extension does not work with Waterfox 55.1.0. 

Was it compatible with any earlier release of 55.x?

**Comment by Kyogre** [comment]
Tried even on Waterfox 54.0.1, but extension still doesn't work
